DataManagementSystem/DB/FileCatalogComponents/DatasetManager.py

```diff
@@ -19,7 +16,7 @@
 
 class DatasetManager:
 
-  _tables = {}
+  _tables = dict
```

Comment:
  Thanks for spotting this bug. What
      are other places with similar bug that you saw in this PR ?
      
        Andrei
      
      On 20/10/15 10:35, andresailer wrote:
    
    
      In DataManagementSystem/DB/FileCatalogComponents/DatasetManager.py:
      > @@ -19,7 +16,7 @@
>  
>  class DatasetManager:
>  
> -  _tables = {}
> +  _tables = dict

      I guess you want "dict()" here and in other cases in this PR.
      —
        Reply to this email directly or view
          it on GitHub.